src/domain/account.ts

```diff
@@ -114,6 +114,7 @@ function createMainAccount(
       accountID: MainAccountID, // main account is unique
       masterXPub: data.masterXPub,
       isReady: true, // always true for main account
+      template: `elwpkh($${MainAccountID})`,
```

Comment:
mmm, we should return the actual descriptor no?

```suggestion
      template: `elwpkh(${data.masterXPub})`,
```